1001-2000/LIT1811Liveso.xml

```diff
@@ -59,7 +66,57 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
    </teiHeader>
    <text xml:base="https://betamasaheft.eu/">
       <body>
-         <ab/>
+        <div type="bibliography">
+            <listRelation>
+                <relation name="lawd:hasAttestation" active="LIT6156OriginProphets" passive="BNFabb195"/>
+            </listRelation>
+               </div>
+            <div type="edition">
+              <note>The division into text parts follows <ref type="mss" corresp="BNFabb195"/>.</note>
+               <div type="textpart" subtype="chapter" xml:id="Chronology">
+                  <label>On the chronology of the Prophets</label>
+               </div>
+               <div type="textpart" subtype="chapter" xml:id="Isaiah">
+                  <label>On Prophet Isaiah</label>
```

Comment:
...the Prophet...?

Reply:
It is plural.

Reply:
Sorry, I meant in front of the names of the prophets: "On the Prophet Isaiah"